programmers-coding-test/sangjun/Implement/시각.js

```diff
@@ -0,0 +1,18 @@
+const rl = require("readline-sync");
+
+const H = parseInt(rl.question("몇시까지 :\n>> "));
+
+let count = 0;
+
+for (let h = 0; h < H + 1; h++) {
+    for (let m = 0; m < 60; m ++) {
+        for (let s = 0; s < 60; s++) {
+            let times = `${h} ${m} ${s}`
+            if (times.match(/[3]/g)) {
```

Comment:
너 코드가 지원이랑 죄다 똑같네? 음.. 

아무튼 정규식은 안쓸수록 좋다. 빠르긴한데, 가독성이 안좋음. 써야할 땐 써야겠지만 이럴 땐 굳이!!
```suggestion
            if (times.includes(“3”)) {
```